src/Pages/yeongseoblee/Main/Feeds/Feed.js

```diff
@@ -0,0 +1,243 @@
+import React, { Component } from "react";
+import CommentItem from "./CommentItem";
+import "./Feed.scss";
+
+class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      comment: "",
+      commentArray: [],
+      clicked: false,
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({ comment: e.target.value });
+  };
+
+  handleEnter = (e) => {
+    if (e.key === "Enter" && this.state.comment.length > 1) {
+      this.appendListItem();
+    }
+  };
+
+  appendListItem = () => {
+    const { commentArray, comment } = this.state;
+    commentArray.push({ cmt: comment });
+    this.setState({
+      commentArray,
+      comment: "",
+    });
+  };
```

Comment:
불변성을 유지하려면 가급적 원본을 사용하지 않고 복사본을 통해 작업하는 것이 좋습니다. 다음 코드를 참고해주세요.

```suggestion
  appendListItem = () => {
    const { commentArray, comment } = this.state;
    this.setState({
      commentArray: [...commentArray, comment],
      comment: "",
    });
  };
```